fabric-item-api-v1/src/main/java/net/fabricmc/fabric/mixin/item/WolfEntityMixin.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.item;
+
+import org.jetbrains.annotations.Nullable;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.entity.passive.WolfEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.FoodComponent;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.Hand;
+
+@Mixin(WolfEntity.class)
+class WolfEntityMixin {
+	@SuppressWarnings("MissingUnique")
+	private static final ThreadLocal<ItemStack> fabric_interactMob_itemStack = new ThreadLocal<>();
```

Comment:
Nit: static final fields should use CONSTANT_CASE

Reply:
itemStack is the name of the local variable. interactMob is the method name where the var is. They should be kept. So the prefix, fabric, should be FABRIC?

Reply:
- Vanilla local variables do not have Yarn names, this is just something generated by Tiny Remapper
- It does not need a Fabric prefix, `@Unique` is enough
- It also doesn't need the exact method name.

Something like `@Unique ThreadLocal<ItemStack> INTERACTION_STACK` works well. See another similar example from this module:
https://github.com/FabricMC/fabric/blob/1681346ec99b3030403d3eedd2773bf2d8d38c4e/fabric-item-api-v1/src/main/java/net/fabricmc/fabric/mixin/item/AbstractFurnaceBlockEntityMixin.java#L36-L37

Reply:
I have fixed this.